core/modules/dashboard/includes/block.update.inc

```diff
@@ -104,10 +104,17 @@ class DashboardUpdateBlock extends Block {
     $panel = array(
       '#theme' => 'dashboard_panel__update',
     );
-    $panel['list'] = array(
-      '#theme' => 'item_list',
-      '#items' => $items,
-    );
+    if (!empty($items)) {
+      $panel['list'] = array(
+        '#theme' => 'item_list',
+        '#items' => $items,
+      );
+    }
+    else {
+      $panel['empty'] = array(
+        '#markup' => '<p class="dashboard-no-updates">' . t('There are no available updates. Your Backdrop site is up-to-date!') . '</p>',
```

Comment:
Why do we need a class added here? ...in fact, why a `<p>`?

```suggestion
        '#markup' => t('There are no available updates. Your Backdrop site is up-to-date!'),
```

Reply:
I don't know, I just thought better to have something rather than nothing, and a class for styling. But I'm not a tyemer so maybe not...